pkg/inject/inject.go

```diff
@@ -254,6 +255,10 @@ func applyAnnotationOverrides(values *l5dcharts.Values, annotations map[string]s
 		values.Proxy.LogLevel = override
 	}
 
+	if override, ok := annotations[k8s.ProxyLogHTTPHeaders]; ok {
+		values.Proxy.LogHTTPHeaders = override
+	}
+
```

Comment:
Just a thought, should we do any logging here in case the value is unknown? For example, a label value of `insecur` will still disable http headers in logs, but perhaps the intention was the opposite. Should we report invalid values to the user somehow?

Reply:
good call.  updated.

```
bin/linkerd upgrade --set proxy.logHTTPHeaders=insecur
× Could not render upgrade configuration: failed to render the template: execution error at (linkerd-control-plane/templates/proxy-injector.yaml:77:12): logHTTPHeaders must be one of: insecure | off
For troubleshooting help, visit: https://linkerd.io/upgrade/#troubleshooting
```